src/App.tsx

```diff
@@ -15,36 +15,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+export enum SortKeys {
+  ALPHABETICAL = 'Alphabetical',
+  LENGTH = 'Length',
+}
+
+export const App: FC = () => {
+  const [sort, setSort] = useState<string>('');
```

Comment:
The initial state of `sort` is set to an empty string. Consider using a default value from the `SortKeys` enum to maintain consistency and avoid potential issues with type safety or logic errors.

Reply:
> The initial state of `sort` is set to an empty string. Consider using a default value from the `SortKeys` enum to maintain consistency and avoid potential issues with type safety or logic errors.

Good point